spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,62 @@
+ require_relative '../spec_helper'
+
+describe UsersController do
+  context 'with authenticated user logged in' do
```

Comment:
If user types in browser `/sign_up` thats GET request and should not be routed to your POST request.